.circleci/config.yml

```diff
@@ -38,7 +38,7 @@ references:
   npm_install: &npm_install
     run:
       name: Install Dependencies
-      command: npm ci --legacy-peer-deps
+      command: npm ci
```

Comment:
I don't think this is needed anymore and it was causing issues with adding peer-dependencies